### ISSUE TITLE
[Documentation] Please consider adding a new entry called "Examples" to https://fpm.readthedocs.io/en/latest/

### ISSUE THREAD

**Issue by rubyFeedback** [jordansissel/fpm#1933]
Hey there Jordan,

fpm is a cool idea. I wanted to add something similar for my "build tools" written
in ruby one day.

Anyway. You provide a lot of high quality documentation at:

https://fpm.readthedocs.io/en/latest/

My question is: would it be possible to add a new distinct subsection called
"Examples" or "More Examples" or so?

My idea with this subsection would be this:

- You can show simple use cases.

- And then, slowly, you show some more complicated examples.

Ideally for .deb, .rpm and perhaps one more (not sure if fpm works
with archlinux pacman but if so an example may be nice to see).

I would like to test these examples on any random package or so,
and I actually learn best from examples, e. g. I try to get the
example to work, and if not then I can read up and look at documentation.
I rarely read the whole documentation; I also don't use local man-pages
on linux for instance, never understood how these make sense in the
modern era.

There do not have to be many examples provided. I think a number
of 10-20 should more than suffice. So these should not necessarily
cover ALL possibilities but just an entry point for newcomers and
semi-vets alone to remember syntax or use cases when they have not
used fpm for some time (or, in case of newcomers, never used it).

It may help the learning process. It may also be useful if you, one
day, decide to "pass the torch" to other maintainers, so that these
could build on the existing knowledge base, including examples
(and once these were added,it is trivial to add more examples
lateron).

At any rate thank you for reading and please proceed with this
issue request in any way you prefer.

**Comment by jordansissel** [comment]
Agreed, it would be helpful to have a section dedicated to examples.
